src/lib.rs

```diff
@@ -65,15 +66,17 @@ impl Toasts {
 
     /// Adds new toast to the collection.
     /// By default adds toast at the end of the list, can be changed with `self.reverse`.
+    /// # Panics
+    ///
+    /// Will panic if after adding a toast the list is empty.
```

Comment:
The function cannot panic as the list literally cannot be empty after we add an element to it(duh)

Reply:
Resolved!